src/events.py

```diff
@@ -311,7 +311,6 @@ def __init__(self, obj):
         conversions = [
             ('estimated_time_of_arrival', util.datetime_from_isoformat),
             ('estimated_time_of_departure', util.datetime_from_isoformat),
-            ('connected_charging_station', str),
```

Comment:
I think this is important? Why is this removed?

Reply:
no, this line was the problem why all "None" values were read in as a string (we hat this issue a while ago). The "connected_charging_stations" are already read in as a str.